Navigator.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { createStackNavigator } from 'react-navigation-stack';
+import { createAppContainer } from 'react-navigation';
+import LoginPage from './src/components/LoginPage.js'
+import RegisterPage from './src/components/RegisterPage.js'
+import HomePage from './src/components/HomePage.js'
+
+const AppNavigator = createStackNavigator ({
```

Comment:
Good idea to have file name match the name of the component to avoid confusion in future.